### PR TITLE
added a plot with includes all replicate stairwayplot estimates

### PR DESCRIPTION
**Pull request by andrewkern**
this PR adds a compound plot of the replicated simulation N(t) estimates from stairwayplot

### CODE REVIEW

**Comment by andrewkern** [comment]
Yes that is in a separate PR. Try running it without the cluster

**Comment by jradrion** [comment]
OK, this works just fine without the cluster.

File: n_t/Snakefile [hunk 1]
```diff
@@ -58,6 +59,16 @@ rule homo_sapiens_Gutenkunst_stairwayplot_plot:
         "homo_sapiens_Gutenkunst/{seeds}.{chrms}.estimated_Ne.png"
     run: plots.plot_stairway_Ne_estimate(input[0], output[0])
 
+def ne_files(wildcards):
+    return(expand("homo_sapiens_Gutenkunst/{seeds}.{chrms}.estimated_Ne.txt",
```
Comment (jeromekelleher) [inline n_t/Snakefile]: Just FYI, the brackets on return aren't necessary here, and are usually omitted.

**Comment by jeromekelleher** [comment]
LGTM, thanks @andrewkern! Merging.

**Pull request merged**